Utils/Dataflow/pyDKB/__init__.py

```diff
@@ -1,5 +1,120 @@
+  storages and keeping them up to date
+
+Dataflow stage
+  Logical step of ETL process, implemented as standalone executable
+  program (worker)
+
+Dataflow stages are standalanoe programs, but can be combined into a
+pipeline by means of Kafka-based supervising program. For details about
+program compatibility with the supervisor please check documentation
+for the Metadata Integration Topology Management System (MInT MS)
+workers [#]_. Worker program can be written in any language; ``pyDKB``
+is intended to simplify this process for Python.
+
+There are three types of stages corresponding three types of ETL
+operations:  *source connector* (data extraction), *processor*
+(transformation) and *sink connector* (load to internal DKB storage).
+Currently ``pyDKB`` library can be used only for *processor* stages, but
+in future versions *connector* stages will also be supported.
+
+Quickstart guide
+----------------
+
+To create simple processor stage application first decide input and
+output data format. In following examples we will work with data in JSON
+format (for the full list of suppurted formats check
+:ref:`pyDKB.dataflow.messages` section of this documentation).
+
+Now let`s start writing example processor ``welcome.py`` and implement
+message handler -- functional part of the stage (operations to be
+performed on data flow units)::
+
+  from pyDKB.dataflow.messages import JSONMessage
+
+  def my_process(stage, message):
+      \""" Single message processing. \"""
+      input_data = message.content()
+      name = input_data.get('name')
+      if name:
+          out_data = {'message': "Welcome, %s!" % name}
+          out_message = JSONMessage(out_data)
+          stage.output(out_message)
+      return True
+
+Function must take two arguments: ``stage`` (stage context object) and
+``message`` (input message, which should be transformed by our stage).
+Message is a smallest data unit in the data flow running through the
+processor, and every message is to be processed independently of
+previous or following ones. ``message.content()`` and
+``JSONMessage(out_data)`` statements are used to decode/encode message
+to/from Python ``dict`` object.
+Message, passed to the function, is taken from the input data flow; to
+write new message(s) to the output data flow,
+``stage.output(out_message)`` is used. It can be used as many times as
+many output messages were generated (or once with the list of messages).
+In our example, messages without key ``'name'`` will produce no output
+messages, so ``stage.output()`` will not be called at all.
+In terms of data flow it means that the input message is filtered out
+and will not reach the *sink connector*..
+
+Boolean return value of ``my_process()`` indicates if the processing was
+successful or not.
+If processing failed (``False`` is returned), produced output messages
+will be dropped to avoid loading into the DKB storages sketchy
+information.
```

Comment:
Word order: "... will be dropped to avoid loading sketchy information into the DKB storages".

Reply:
Fixed.